src/app/tests/suites/certification/Test_TC_OO_2_4.yaml

```diff
@@ -238,6 +238,21 @@ tests:
       PICS: OO.S.C00.Rsp
       command: "Off"
 
+      # We get an acknowledgment that the command off above is recieved, but it takes
+      # a few milliseconds for the off command to actually turn it off. Without this
+      # delay the preceding reboot command might happen before the command is fully
+      # processed resulting in flaky test failure that comes from the readAddribute
+      # verification after the reboot.
+    - label: "Wait for send Off command to take affect"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      # We know the lockout value, but it's in seconds. Previous read from UserCodeTemporaryDisableTime verified that
```

Comment:
What does UserCodeTemporaryDisableTime have to do with anything?

Reply:
Copy and paste error. Thanks for catching this